README.md

```diff
@@ -1,2 +1,60 @@
 # admission-trial
- 
+ # Portfolio
+
+This is my Microverse project called "Portfolio" (Mobile and Desktop version) built using HTML and CSS only.
+
+Built with: HTML and CSS
+
+Linters such as :
+
+1. Sytle Lint
+2. WebHint
+3. Lighthouse
+
+# Getting Started
+
+- Create a new repo
+- Create a new branch according to the GitHub flow rules.
+- Add linters
+- Add a git ignore file
+- Add HTML & CSS files
+- Open a pull request
+
+### Prerequisites
+
+- Git. Which is a open source distributed version control system designed for source code management.
+- A text editor (e.g Visual Studio Code, Vim.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+
+### Setup
+
+- Using git, clone the code to your machine or download a Zip File.
+
+👤 **Authors**
+
+👤 Emmanuel Hagan
+
+- GitHub: (https://github.com/hagann109)
+- Twitter: (https://twitter.com/emmahagan199)
+- LinkedIn:(https://linkedin.com/inemma199)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+- Contributions, issues, and feature requests are welcome!
+
+- Feel free to check the [https://github.com/hagan199/my-portfolio/issues].
+
+https://github.com/hagan199/my-portfolio/issues
+## Acknowledgments
+
+- Microverse
+- Inspiration
+
+
+## 📝 License
+
+
+- This project is [MIT](./MIT.md) licensed.--
```

Comment:
Please kind edit info with the proper detail

Reply:
Kindly check how a professional README is constructed. Your Getting Started, Prerequisite, Contrbuting, and Setup sections should carry your audience along on what is needed to run the application and how to go about setting up a local version.

Reply:
Thank  you for the advice